_posts/events/2019-09-25-europhras.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+categories: [event]
+event_name: "Computational and Corpus-based Phraseology (Europhras 2019)"
+event_url: "http://www.lexytrad.es/europhras2019"
+event_location: "Malaga, Spain"
+event_submission_date: "2019-05-15"
+siglex_endorsed: yes
+---
+Computational and Corpus-based Phraseology (Europhras 2019)
+
+Malaga, 25-27 September 2019
```

Comment:
You probably want two spaces at the end of this line so that the URL shows up on its own line.

Reply:
Done